unified-codes/apps/web/src/components/templates/Explorer.tsx

```diff
@@ -14,7 +14,13 @@ import { ExplorerActions } from '../../actions';
 import { IExplorerData, IState } from '../../types';
 import { ExplorerSelectors } from '../../selectors';
 
+import { withStyles } from '@material-ui/core/styles';
+import { ITheme, overflow } from '../../muiTheme';
+
+const FOOTER_HEADER_HEIGHT = 300;
+
 export interface ExplorerProps {
+  classes?: IEntityBrowserClasses;
```

Comment:
Bit confused about prop drilling classes. Could this be something defined on a per component basis rather than passed down?

Reply:
yes, except that `EntityBrowser` is wrapping so many components and I want the styles in the web, which is why I passed down in classes. another discussion point?

Reply:
Yup, understand your point. Something to be discussed in a dedicated issue, as you say.